cmd/libsnap-confine-private/cleanup-funcs-test.c

```diff
@@ -142,6 +143,31 @@ static void test_cleanup_close(void)
 	g_assert_cmpint(fd, ==, -1);
 }
 
+static void test_cleanup_deep_strv(void)
+{
+	/* It is safe to use with a NULL pointer */
+	sc_cleanup_deep_strv(NULL);
+
+	char **argses = NULL;
+	/* It is OK if the pointer value is NULL */
+	sc_cleanup_deep_strv(&argses);
+	g_assert_null(argses);
+
+	/* It is OK if the string pointers are NULL */
```

Comment:
the comment is a bit unclear as NULL is the terminator for the vector if I understand the code

Reply:
Good point, I'll clear this comment up